types/modules/producer.d.ts

```diff
@@ -7,7 +7,7 @@ declare class ProducerError extends Error {
   constructor(error: { name: string; message: string });
 }
 
-interface PublishOptions extends amqp.Options.Publish {
+interface ProduceOptions extends amqp.Options.Publish {
```

Comment:
We need to decide, which is of the ones we shoul depricate: either `Publish/Subscribe` or `Produce/Consume`

Reply:
It should be produce / consume, this is how RabbitMQ clients call it.
Maybe let's deprecate publish / subscribe in our next version?